addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -137,7 +138,10 @@ function factory(dependencies) {
          * @param {Event} ev
          */
         onMouseLeaveTopBarThreadName() {
-            this.update({ isMouseOverThreadName: false });
+            if (!this.thread || this.thread.model !== 'mail.channel') {
+                return;
+            }
+            this.update({isMouseOverThreadName: false});
```

Comment:
I believe this diff is uncessary

Reply:
So what about ```onMouseEnterTopBarThreadName``` would it be consistent to change it there too ?

Reply:
I don't understand why do you want to change anything in regard to the rename in the top bar?
Apart from the focus of course, but the rest should be working fine after rebase?

Reply:
Ok got it